Whatsapp_Automated_Messages/ReadME.md

```diff
@@ -0,0 +1,17 @@
+AIM
```

Comment:
Format this readme
- All headings need to have levels. Level 2 (##)

Reply:
Where am I support to add the level??

Reply:
```## Script name```